backend/src/main/java/heartbeat/service/report/GenerateReporterService.java

```diff
@@ -245,26 +251,30 @@ public synchronized ReportResponse generateReporter(GenerateReportRequest reques
 				}
 			}
 		});
-		generateCSVForMetric(reportResponse, request.getCsvTimeStamp());
 
-		saveReporterInHandler(reportResponse, request.getCsvTimeStamp());
 		return reportResponse;
 	}
 
 	private FetchedData fetchOriginalData(GenerateReportRequest request, List<String> lowMetrics) {
 		FetchedData fetchedData = new FetchedData();
 
 		if (lowMetrics.stream().anyMatch(this.kanbanMetrics::contains)) {
+			if (request.getJiraBoardSetting() == null)
+				throw new BadRequestException("Jira board setting is null.");
```

Comment:
all exception/failed message should be started with `Failed to do ....`

Reply:
Already update exception message which will follow our convention.